content/docs/command-reference/repro.md

```diff
@@ -132,7 +134,7 @@ up-to-date and only execute the final stage.
   to. Use `dvc dag <target>` to show the parent pipeline of a target.
 
 - `-P`, `--all-pipelines` - reproduce all pipelines for all `dvc.yaml` files
-  present in the DVC project.
+  present in the <abbr>DVC project</abbr>.
```

Comment:
Why this change? Completely unrelated to the PR. Also, we don't use `<abbr>` in Options (it's too much to have tooltips here).
```suggestion
  present in the DVC project.
```

Reply:
Done. 
> we don't use <abbr> in Options

I forgot this. Thanks.